refinery/tool_manager/tests.py

```diff
@@ -637,6 +636,39 @@ def test_workflow_list_not_enough_inputs(self):
             )
             self.assertEqual(ToolDefinition.objects.count(), 0)
 
+    def test_visualization_annotation_extra_dirs_valid(self):
+        with open(
+            "{}/visualizations/LIST_visualization_good_extra_directories.json"
+            .format(TEST_DATA_PATH)
```

Comment:
would you like to add "visualizations" to TEST_DATA_PATH? It seems to be repeated a lot.